package.json

```diff
@@ -31,6 +31,7 @@
     "@next/bundle-analyzer": "^12.0.8",
     "@next/mdx": "^12.2.3",
     "@rjsf/core": "^4.1.1",
+    "@types/mongoose-delete": "^0.5.9",
```

Comment:
This should be move into `devDependencies`

Reply:
done